### PR TITLE
Don't drop to half the vsync rate when we can't keep up

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by jherico** [comment]
https://app.asana.com/0/27650181942747/72197876981925

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -984,6 +989,17 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer) :
...
+    _idleTimer = new QTimer(this);
+    connect(_idleTimer, &QTimer::timeout, [=] {
+        idle(usecTimestampNow());
+    });
+    connect(this, &Application::beforeAboutToQuit, [=] {
+        disconnect(_idleTimer);
+    });
+    // Setting the interval to zero forces this to get called whenever there are no messages
+    // in the queue, which can be pretty damn frequent.  Hence the idle function has a bunch 
+    // of logic to abort early if it's being called to often.
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: "too often"

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -4777,11 +4831,11 @@ void Application::updateDisplayMode() {
         foreach(auto displayPlugin, standard) {
             addDisplayPluginToMenu(displayPlugin, first);
             // This must be a queued connection to avoid a deadlock
-            QObject::connect(displayPlugin.get(), &DisplayPlugin::requestRender, [=] {
-                postEvent(this, new LambdaEvent([=] {
-                    paintGL(); 
-                }), Qt::HighEventPriority);
-            });
+            //QObject::connect(displayPlugin.get(), &DisplayPlugin::requestRender, [=] {
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: dead code? can we delete?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 908e153c3609d750917293570bdee25c6527c2e6 available at: https://gist.github.com/12a5fe7888c485776598<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7103.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7103.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: df225afd058f6029cf97b3946f66aa289a5fd9ca available at: https://gist.github.com/12a5fe7888c485776598<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7103.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7103.dmg

**Comment by jherico** [comment]
test this please

**Comment by cozza13** [comment]
tested this and was able to see a render rate in areas between 37 and 75. passed

**Comment by hifi-gustavo** [comment]
Build console output for commit: df225afd058f6029cf97b3946f66aa289a5fd9ca available at: https://gist.github.com/12a5fe7888c485776598<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7103.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7103.dmg

**Pull request merged**